tests/e2e/specs/monitor-home.spec.js

```diff
@@ -0,0 +1,84 @@
+const { test, expect } = require('@playwright/test');
+const { HomePage } = require('../pages/homePage');
+const { LandingPage } = require('../pages/landingPage');
+const { LoginPage } = require('../pages/LoginPage');
+
+test.describe('Monitor home page', () => {
+  test.skip(
+    ({ browserName }) => browserName !== 'chromium' && browserName !== 'webkit',
+    'Chromium and webkit only!'
+  );
+
+  test.beforeEach(async ({ page }) => {
+    const landingPage = new LandingPage(page);
+    await landingPage.open();
+    await landingPage.goToLogin();
+
+    const loginPage = new LoginPage(page);
+    await loginPage.login(
+      process.env.TESTACCOUNT_EMAIL,
+      process.env.TESTACCOUNT_PASSWORD
+    );
+  });
+
+  test('Verify elements exist in DOM', async ({ page }) => {
+    const homePage = new HomePage(page);
+
+    expect(page.url()).toContain('https://monitor.firefox.com/user/dashboard');
```

Comment:
I wonder if this could be tweaked to read from an env var or something to allow you to run these tests against localhost or staging or heroku, etc.